sdk/monitor/monitor-query/test/public/logsQueryClient.spec.ts

```diff
@@ -4,31 +4,34 @@
 import { assert } from "chai";
 import { Context } from "mocha";
 import { env } from "process";
-
+import { createRecorderAndLogsClient, RecorderAndLogsClient } from "./shared/testShared";
+import { Recorder } from "@azure-tools/test-recorder";
 import { Durations, LogsQueryClient, QueryBatch } from "../../src";
-import { runWithTelemetry } from "../setupOpenTelemetry";
-
-import {
-  assertQueryTable,
-  createTestClientSecretCredential,
-  getMonitorWorkspaceId,
-  loggerForTest
-} from "./shared/testShared";
+// import { runWithTelemetry } from "../setupOpenTelemetry";
```

Comment:
```suggestion
```

Reply:
I have disabled the tests using opentelemetry-exporter for data ingestion - bcz there is a bug in that package - also bcz it doesn't support browser. We don't really need tests to do the data ingestion. So commenting it out for now.